contracts/libraries/UserLib.sol

```diff
@@ -113,6 +113,11 @@ library UserLib {
     address delegateUser;
   }
 
+  struct LinkedAccount {
+    address wallet;
```

Comment:
Keep the name of the field consistent with other structs. I don't think we call it `wallet` anywhere.